lib/archivist/index.js

```diff
@@ -526,7 +526,15 @@ function mgetAggregate(queries, retriever, cb) {
 		async.forEachSeries(
 			Object.keys(queries),
 			function (queryId, callback) {
-				retriever(queries[queryId], function (error, data) {
+				const query = queries[queryId];
+
+				if (!query.options) {
+					query.options = {};
+				}
+
+				query.options = Object.assign({}, options, query.options);
```

Comment:
Rather than mutating the `query` object (which archivist does not own), why not do this instead:

```js
const queryOptions = query.options ? Object.assign({}, options, query.options) : options;

retriever(query, queryOptions, function (error, data) {
```